docker/docker-compose.yaml

```diff
@@ -0,0 +1,16 @@
+version: '2'
```

Comment:
I got errors trying to run docker compose on this file and it turns out that ubuntu has an older (pre 1.6) version of docker compose that doesn't support [versioning](https://docs.docker.com/compose/compose-file/#/versioning). It might be useful to either provide a version 1 docker compose file, or put in the deployment guide that this requires [docker-compose > 1.6](https://docs.docker.com/compose/compose-file/#/version-2).